monero-outreach-docs/translations/pt-br/storyboard/blockchain_gender_pt_br.md

```diff
@@ -0,0 +1,78 @@
+
+A maioria de nós não percebe que nós não temos controle do nosso dia-a-dia e que muitos itens essenciais são controlados por outras pessoas e não há como evitar isso. Nosso sistema foi projetado dessa forma para que os “poderosos” possam controlar os “vulneráveis”. Acredito que esta "Revolução da Criptomoeda" surgiu da necessidade de tentar recuperar o poder, resistindo à censura. As pessoas deveriam ter autoridade para administrar suas finanças, bem como distribuir a riqueza dentro das sociedades por meio desse sistema ponto a ponto de dinheiro digital, sem que um intermediário assuma o controle.
+
+**Você esteve envolvida no projeto Monero por um bom tempo. Você pode compartilhar alguns detalhes sobre o seu envolvimento, sobre quando e como começou e o que faz?**
+
+Eu ingressei no Monero no final de 2016. Eu conheci muitas moedas alternativas e percebi que Monero não era como as outras. Comecei a trabalhar em projetos relacionados ao Monero no início deste ano, criando o primeiro Monero Meetup em Nova York. Na época só víamos encontros de blockchain ou bitcoin, então nosso grupo - incluindo [@chowbungaman](https://twitter.com/chowbungaman) e [Vik Sharma](https://www.youtube.com/watch?v=joAZQXp69Kw), fundador do [Cake Wallet](https://cakewallet.io/), a primeira Carteira Monero para iOS - decidiu iniciar um Monero Enthusiast Meetup. Tivemos um público maravilhoso e o grupo continua a prosperar.
+
+Devido a esse grande sucesso, planejamos iniciar um grupo na Filadélfia. Eu também estive envolvida no desenvolvimento e no planejamento do After Party do  NYC Monero Speakeasy Consensus em maio de 2018, junto com @chowbungaman e Vik. Este evento foi uma ótima maneira de incentivar a comunidade a usar o Monero como forma de pagamento e acabou sendo um grande sucesso.
+
+Mais recentemente, estive envolvida nos bastidores como gerente de operações/coordenadora de um canal no YouTube chamado “[Monero Talk w/ @chowbungaman](https://www.youtube.com/channel/UC3Hx81QYLoEQkm3vyl4N4eQ)”. É um canal sobre todas as coisas relacionadas ao Monero. Queremos fornecer e complementar informações envolvendo o Monero.
+
+Como atualmente tudo é mais voltado ao Bitcoin, decidimos criar este canal para as pessoas que estão interessadas no Monero e até agora ele está indo muito bem. Tivemos grandes visitantes e nos disseram que ele é muito informativo. Estamos tentando expor a todos as informações relacionadas ao Monero, um episódio de cada vez.
+
+**O que atraiu você em particular sobre o Monero que fez você querer se envolver de uma maneira mais substancial?**
+
+Ele tem todos os parâmetros que o Bitcoin está tentando alcançar: é privado, não rastreável e fungível. Essas são as qualidades perfeitas de uma criptomoeda criada para atingir sua meta de descentralização e que poderia ser usada como uma moeda digital ponto a ponto.

```

Comment:
Use "não-rastreável" for [consistency](https://github.com/monero-ecosystem/monero-translations/blob/master/terminology-guides/portuguese-br-terminology.md).

Also "ponto-a-ponto" as stated earlier.